calculator.py

```diff
@@ -1,14 +1,14 @@
 def add(x, y):
-    pass
+    return x+y
 
 def subtract(x, y):
-    pass
+    return x-y
 
 def multiply(x, y):
-    pass
+    return x*y
 
 def divide(x, y):
-    pass
+    return x/y
```

Comment:
Will aprove when you fix division by zero conflict. Just kidding haha. Aproved!!!